composer.json

```diff
@@ -47,7 +47,7 @@
 		"optimize-autoloader": true,
 		"process-timeout": 0,
 		"platform": {
-			"php": "7.4"
+			"php": "7.4.30"
```

Comment:
Let platform be an existing PHP version.

Reply:
From the [docs](https://getcomposer.org/doc/06-config.md#platform), it seems `"7.4"` is valid.

If it isn't though, wouldn't we want at `"7.4.0"`? Any `7.4.x` version is supported and we want to make sure we grab deps that are compatible with all of them.

Reply:
This is not a requirement. It is a fake value that Composer uses.
It has nothing to do with your actual PHP version.
I leave it 100% up to you. I am not a user.

Reply:
I set it to match the production environment.
https://www.php.net/ChangeLog-7.php
v7.4.30 was released a year ago.

Reply:
I hope there is no Composer package that is not compatible with a higher path version of 7.4.x!

Reply:
The issue is for a _user_ running 7.4.0, if a dep release requires 7.4.1 it shouldn't be installed.

Reply:
All right.
Now it is 0.